content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Python"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Rust"}}
+
+**SDK: Outbound HTTP**
+
+Local: Yes
+
+Cloud: Yes
+
+Documentation: [https://developer.fermyon.com/spin/rust-components.md#sending-outbound-http-requests](https://developer.fermyon.com/spin/rust-components.md#sending-outbound-http-requests)
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http](https://github.com/fermyon/spin/tree/main/examples/http-rust-outbound-http)
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local: Yes
+
+Cloud: Soon
+
+Documentation: [https://developer.fermyon.com/spin/kv-store.md](https://developer.fermyon.com/spin/kv-store.md)
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/rust-key-value](https://github.com/fermyon/spin/tree/main/examples/rust-key-value)
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local: Yes
+
+Cloud: Yes (non-hosted)
+
+Documentation: Soon
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/rust-outbound-mysql](https://github.com/fermyon/spin/tree/main/examples/rust-outbound-mysql)
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local: Yes
+
+Cloud: Yes (non-hosted)
+
+Documentation: Soon
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/rust-outbound-pg](https://github.com/fermyon/spin/tree/main/examples/rust-outbound-pg)
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local: Yes
+
+Cloud: Yes (non-hosted)
+
+Documentation: [https://developer.fermyon.com/spin/rust-components.md#storing-data-in-redis-from-rust-components](https://developer.fermyon.com/spin/rust-components.md#storing-data-in-redis-from-rust-components)
+
+Code Examples: [https://github.com/fermyon/spin/tree/main/examples/rust-outbound-redis](https://github.com/fermyon/spin/tree/main/examples/rust-outbound-redis)]
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
```

Comment:
Either "custom triggers" (for general behaviour) or "the cron trigger" (for custom triggers with specific support).

I would suggest not talking about cron in this context - we have, I think, a timer sample, but not a working cron trigger that people can use.